### DISCUSSION TITLE
Start counting the page number 1 on page 2 and skip the first page in total pages.

### DISCUSSION THREAD

**Discussion by wesleigomes** [QuestPDF/QuestPDF#87] [Q&A] [answered]
Hi,

I would like to start counting the page number 1 on page 2 and skip the first page in total pages (Example image).
My first page is a cover, without relevant content.

![example](https://user-images.githubusercontent.com/16633920/147958514-c0e1a865-0ceb-4b3d-926e-9b99cec676e0.png)

Can anyone help me?

**Comment by MarcinZiabek** [comment, marked as answer]
Hello 😀 I am afraid that QuestPDF currently does not support this particular requirement. Most likely it will be available along with the `Dynamic Component` feature. It is designed to give more freedom and page-aware context to the developer.

Right now, you can generate two separate PDF documents using QuestPDF (cover page and content) and then merge them together using other free libraries. It sounds like a good plan, doesn't it? 😁

**Reply by wesleigomes** [reply]
Thank you! Yes, it can be a solution.

**Comment by DawidRobertNowak** [comment]
Hello,

We're closing this thread as the requested functionality is now supported natively, and you can learn more [here](https://www.questpdf.com/api-reference/text/page-numbers.html). Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new discussion or issue.

Thank you!
